config/travis/.solr_wrapper

```diff
@@ -0,0 +1,7 @@
+# Place any default configuration for solr_wrapper here
+# version: 6.1.0
```

Comment:
Since we're using the latest version of solr, is there any reason to do caching?

Reply:
technically the error message I was getting suggested the quota was bytes based, rather than number of requests: 10GB per 24 period from a given IP address, which presumably many travis builds share an IP address. 

Same same for what's being discussed, but there's been so much confusion on this I figured I'd leave some details here in the log.

Reply:
Also, the error message I was getting suggested an *.apache.org host was being hit, and enforcing this quota, not actually a mirror at all. If it was actually hitting a mirror it might not be a problem, but I think there are other confusing reasons the mirror(s) can't be/aren't used in some cases. 

The whole thing is very confusing to me, and I'm not myself sure what the right fix is at the right level of our software stack -- I think really solr_wrapper needs to _stop_ using apache servers to download solr, since this seems to be something apache doesn't like. But there seem to be confusing reasons that this is not feasible.

Reply:
@awead even on the mirrors? I've only ever seen the throttling on the archive machines, which are not mirrored.

Reply:
Our repos seem to be using the archive machines, not the mirrors, so would benefit from caching, no?  I may be confused, there are a lot of moving parts here. But I'm definitely getting archive machine quota throttling on at least some repos, like sufia. 

I guess if it hasn't happened on the AF repo yet, maybe it doesn't effect this one for some reason?